src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/request/RequestStatus.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.request;
+
+public enum RequestStatus {
+    OPEN,
+    PROCESSED,
```

Comment:
PROCESSED isn't in the spec (at least for now) so I think it should be removed from here.